README.adoc

```diff
@@ -11,81 +11,79 @@ The primary goal of the https://projects.spring.io/spring-data[Spring Data] proj
 
 The Spring Data Neo4j project aims to provide a familiar and consistent Spring-based programming model for integrating with the https://neo4j.com/[Neo4j] Graph Database.
 
-=== Features
+== Code of Conduct
```

Comment:
Let's move the CI badges to a dedicated file (see https://github.com/spring-projects/spring-data-mongodb/blob/master/CI.adoc). Same goes for the CI build instructions.